sdk/hdinsight/arm-hdinsight/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/arm-hdinsight",
   "author": "Microsoft Corporation",
   "description": "HDInsightManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "0.16.1",
+  "version": "0.17.1",
```

Comment:
Major version 0 is considered as a "preview" version in general in the JS community. Can we change the version here to 1.0.0 after getting someone from the service team to test the package?